### DISCUSSION TITLE
Gettig error in npm install

### DISCUSSION THREAD

**Discussion by Sarthakaga15** [Ride-The-Lightning/RTL#973] [Q&A] [answered]
Hi, I am getting errors while running the server on npm run server before I was getting at npm install, I have tried re cloning repo and still, I am getting this I am running lnd and bitcoind already in the background.  Please need your suggestion on what I am doing wrong .
![rtl](https://user-images.githubusercontent.com/68947969/156917010-2e64dd85-3037-45a0-b031-9b9b10e813aa.png)
![rtl 1](https://user-images.githubusercontent.com/68947969/156917016-0e14a24c-5bd0-4df7-8b67-0852ae25caa3.png)

**Comment by ShahanaFarooqui** [comment, marked as answer]
Did you update configuration in RTL_Config.json file? These values look like default.

https://github.com/Ride-The-Lightning/RTL#prep

**Comment by Sarthakaga15** [comment]
Hi @ShahanaFarooqui ,Okay, I'll update and retry. Actually, I cloned and ran RTL in a different folder, but all of these commands performed, and the application ran without altering this file, as my LND isn't installing there since it doesn't find GO there, despite the fact that it was installed. Pardon for inconvenience; all of this is happening because my Linux and Windows node and go installations are colliding occasionally processes don't start because they are already running, producing fresh errors every time. I'll fix these problems as soon as possible, or else I'll install a separate Linux subsystem from windows.
I'm also learning the tech stack needed for projects, particularly interested in the RTL quickpay extension and WebLN support. If you have any recommendations to help me prepare and learn better, please let me know.

**Reply by ShahanaFarooqui** [reply]
RTL quickpay is built with javascript, jquery and webpack.
